notes/TwoArgumentsTwoContinuationsDecompiled.txt

```diff
@@ -0,0 +1,184 @@
+/*
+ * Decompiled with CFR 0.152.
```

Comment:
I have to say in case people want to look more at the logic, even though this is closer to how our code looks like, I found the outcome from `Procyon` decompiler easier to read and understand.